rpcserver.go

```diff
@@ -1148,10 +1148,17 @@ func (s *rpcServer) RecoverAccounts(ctx context.Context,
 
 	s.recoveryMutex.Lock()
 	if s.recoveryPending {
-		defer s.recoveryMutex.Unlock()
+		s.recoveryMutex.Unlock()
```

Comment:
nit: alternatively we could just use an atomic int to avoid the (slightly complex) locking logic.